wireless.lua

```diff
@@ -148,14 +147,24 @@ local function worker(args)
             if popup_signal then
                 signal = "├Strength\t"..signal_level.."\n"
             end
+
+						metrics_down = ""
+						metrics_up = ""
+						if popup_metrics then
+							local tdown    = os.capture("ntotal "..interface.." d")
+							local tup    = os.capture("ntotal "..interface.." u")
+							metrics_down = "├DOWN:\t\t"..tdown.."\n"
+              metrics_up = "├UP:\t\t"..tup.."\n"
+						end
+
```

Comment:
Replace tabs with spaces, please, because there is no tabs elsewhere :)

Reply:
converted to spaces now 👍